platform/posix/transport/src/plaintext_posix.c

```diff
@@ -198,7 +159,7 @@ int32_t Plaintext_Recv( NetworkContext_t * pNetworkContext,
         bytesReceived = 0;
     }
 
-    if( ( selectStatus > 0 ) && ( bytesReceived == 0 ) )
+    if( ( pollStatus > 0 ) && ( bytesReceived == 0 ) )
```

Comment:
(Nit) A comment can be added to mention that a zero value return from `recv` represents closure of TCP connection by the peer. I know that this comment exists within this branch, but it doesn't explicitly mention about it being related to `recv` call